trpcultivate_phenotypes/tests/src/Traits/PhenotypeImporterTestTrait.php

```diff
@@ -183,69 +183,77 @@ protected function createTestFile($details) {
 
     // Set Defaults.
     $details['extension'] = @$details['extension'] ?: 'txt';
-    $details['mime'] = @$details['mime'] ?: 'text/tab-separated-values';
     $details['filename'] = @$details['filename'] ?: 'testFile.' . uniqid() . '.' . $details['extension'];
+    $details['mime'] = @$details['mime'] ?: 'text/tab-separated-values';
     $details['is_temporary'] = @$details['is_temporary'] ?: FALSE;
     $details['content'] = @$details['content'] ?: ['string' => uniqid()];
 
-    // Determine the fullpath to test files for use later
-    if (array_key_exists('file', $details['content'])) {
-      $path_to_fixtures = __DIR__ . '/../Fixtures/';
-      $full_path = $path_to_fixtures . $details['content']['file'];
-    }
-
+    // Set directory.
     $directory = ($details['is_temporary']) ? 'temporary://' : 'public://';
-    $uri = $directory . $details['filename'];
+    $file_uri = $directory . $details['filename'];
 
+    // Create file object.
     $file = File::create([
       'filename' => $details['filename'],
       'filemime' => $details['mime'],
-      'uri' => $uri,
+      'uri' => $file_uri,
       'status' => 0,
     ]);
 
-    // Set the size of the file.
-    // This is usually used if the file is empty in which case this is 0
-    if (isset($details['filesize'])) {
-      $file->setSize($details['filesize']);
-    }
-    else if (array_key_exists('file', $details['content'])) {
-      $size = @filesize($full_path);
-      // Set size unless there was an error.
-      $this->assertNotFalse($size, 'Unable to determine size of test file: ' . $full_path);
-      $file->setSize($size);
-    }
+    // Reference file attributes:
+    $file_id = $file->id();
+    $file_uri = $file->getFileUri();
```

Comment:
Just out of curiosity, why does $file_uri need to be redefined here after it was defined on line 193? Shouldn't it still be the same or is it possible that it changed?